src/services/transaction/TransactionMaterialService.ts

```diff
@@ -11,10 +11,34 @@ export class TransactionMaterialService extends AbstractService {
 	 * @param hash `BlockHash` to make call at
 	 */
 	async fetchTransactionMaterial(
-		hash: BlockHash
+		hash: BlockHash,
+		noMeta?: string
```

Comment:
Would suggest making this a boolean, so then the `if` statement can be rewritten as `if (noMeta)`, which I think is cleaner.